src/constants/emoji.ts

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-export default {
+const Emoji = {
```

Comment:
Have you checked if this makes a difference to the compiled code? I'm pretty sure this is functionally the same as before.

Reply:
I'm not sure there's any difference between the two ways of writing this. What's the benefit?

Reply:
I was under the impression that when we declared these objects, all of their values defaulted to the type of the value we assigned. If we declare as `const`, we know that the object can't change and thus issues with comparing these values to a string literal type are gone since the value can only be the specified string literal.

I might be wrong in this belief, maybe the issue is actually just the typing of the object itself, like in the `groups.ts` case.

Reply:
`const` in JS is like final in Java, so the contents of Emoji can change in both cases. When you use `export default`, I don't think the reference can change anyway because it doesn't have a name.